### PR TITLE
Feature/videocategory

### PR DESCRIPTION
**Pull request by dent1p**

### CODE REVIEW

**Comment by mnk87** [comment]
If i try to delete a category that is already linked to one or more videos i get a laravel error that a constraint fails, do we want to leave it that way for now or do we want that fixed?

Other than that it looks good.

**Comment by HonkingGoose** [comment]
The database is ensuring the integrity of the data, it won't delete a record that is still linked with a foreign key.

Maybe a proper error message: "You're trying to delete a category that still has videos attached." is more helpful than erroring out with a Laravel boilerplate message.

File: database/migrations/2020_02_05_145603_create_videos_table.php [hunk 1]
```diff
@@ -25,6 +25,9 @@ public function up()
             $table->boolean('suitable_for_kids');
             $table->string('created_by');
         });
+
```
Comment (HonkingGoose) [inline database/migrations/2020_02_05_145603_create_videos_table.php]: Remove unnecessary white-space please.

File: database/seeds/VideosTableSeeder.php [hunk 2]
```diff
@@ -12,12 +14,15 @@ class VideosTableSeeder extends Seeder
...
 
-        //for now, we can use this to insert a couple of entries in the database
-
-        //\App\Video::truncate();
+       //Thumbnail watch_video factory
 
         (new Faker\Generator)->seed(123);
 
         factory(App\Video::class, 19)->create();
+
```
Comment (HonkingGoose) [inline database/seeds/VideosTableSeeder.php]: Remove unnecessary white-space please.

**Review by HonkingGoose — CHANGES_REQUESTED** [review]

**Review by HonkingGoose — DISMISSED** [review]

**Review by mnk87 — DISMISSED** [review]

**Review by HonkingGoose — APPROVED** [review]

**Pull request merged**